### DISCUSSION TITLE
Golf course

### DISCUSSION THREAD

**Discussion by TanGitDev** [osmandapp/OsmAnd#19344] [Ideas]
I recently added a golf course with more than 50 bunkers (golf=bunker) and 36 greens (golf=green) but it happens that none of that is displayed on OSMAND render while these features appears on OSM.org online render.
Can you implement this feature for better rendering?
Thanks

**Comment by DmitryAlexei** [comment]
@TanGitDev have you enabled **More details** option in Configure map > details? 
You may also need to enable **Live updates** (see https://osmand.net/docs/user/personal/maps#osmand-live) or wait for new map generation at the beginning of April. 

For reference, you can go to 42.68057 23.52787 and see, that golf=bunker with natural=sand are rendered as sand regions (not as POI).


<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/116079541/7bc417dd-d663-4449-9891-119265768709"  width="300" height="auto"/> </td>
  </tr>
</table>

**Reply by TanGitDev** [reply]
@DmitryAlexei 
Hello, indeed I just foud out that adding natural=sand to the tag enables the render. That might do the trick for bunkers.
I might also add natural=grass to putting greens but then I would not be able to differentiate between green, fairway and meadow.
I found the OSM.org render quite good if you want to get some inspiration.

**Reply by tommycrock** [reply]
Hi @TanGitDev 
Please just use the golf tags and don't add natural=sand or landuse=grass or other such tags to golf course features to get them to look the way you want on a particular renderer. OSM is a geographic database and those things have a different meaning. This is mentioned explicitly on the wiki https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dgolf_course#Features_within_a_golf_course
If there's a problem with the way osmand displays certain golf course features (I'm not sure there is) then it's up to osmand to fix. Cheers

**Reply by TanGitDev** [reply]
@tommycrock
If natural=sand SHOULD not be used in combination with golf=bunker then it should be explicitely mentionned on the bunker page and not only on golf_course page.

**Reply by tommycrock** [reply]
Good idea, I thoroughly agree 👍

**Reply by lenaloos** [reply]
Your addition of over 50 bunkers and 36 greens to the golf course merits visibility on OSMAND render. After searching very deeply, I have come to the conclusion that you can get some benefit with the help of this [content](https://golfflipper.com/what-degree-loft-is-a-pitching-wedge/) and if it is not enough for you, you can do more about it on the other side.

**Reply by mikehgentry** [reply]
The above comment appears to be AI generated spam.

Not sure if/how that should be reported?

@DmitryAlexei @yuriiurshuliak 

edit: turns out you can report spam accounts through the user's profile, which I have now done.
